kuma/static/js/stripe-subscription.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+    var stripeConfigElement = document.getElementById('stripe-config');
+    var stripeSection = document.querySelector('.stripe-subscription');
+    var stripeForm = document.getElementById('stripe-form');
+
+    if (
+        stripeSection.scrollIntoView &&
+        location.search.indexOf('has_stripe_error') !== -1
+    ) {
+        stripeSection.scrollIntoView({behavior: 'smooth'});
+    }
+
+    if (!stripeForm) {
```

Comment:
This unfortunately got a little confusing, since the script is loaded for both the not-yet-completed form and the completed form. The stripe section exists in either case, but the form only exists when the subscription is not set up yet. Hence the order should be correct.

Reply:
Added comments to make this more clear.